Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -245,8 +398,13 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
     if (m_Card)
     {
       u32 size = req.bsize * req.blocks;
+      u64 address = req.arg;
+      if (m_Status & CARD_SDHC)
```

Comment:
Can you move this to a named method (like, I dunno, `GetAddress` or whatever sounds feasible in this context) and call it instead of repeating this `if` multiple times? A descriptive name can help in documenting the code without adding extra comments.